plugin/pkg/admission/defaulttolerationseconds/admission.go

```diff
@@ -82,7 +82,19 @@ func (p *Plugin) Admit(attributes admission.Attributes) (err error) {
 
 	toleratesNodeNotReady := false
 	toleratesNodeUnreachable := false
-	for _, toleration := range tolerations {
+	for i, toleration := range tolerations {
+		// replace alpha toleration key with GA key (remove alpha)
+		if toleration.Key == algorithm.DeprecatedTaintNodeNotReady {
+			toleration.Key = algorithm.TaintNodeNotReady
```

Comment:
this will register as a removal of a toleration on an update, which is not allowed

Reply:
OK, I leave this logic as is so the existing tolerations are untouched, the newly added unit tests will verify this behavior.

Reply:
I expected an attempt to add the new toleration, leaving the old toleration in place (though doing it here wouldn't address existing pods present in the system, only ones that got updated)